### ISSUE TITLE
Cannot install from original CDs

### ISSUE THREAD

**Issue by guest73** [vcmi/vcmi#1819]
**Describe the bug**
When installing with vcmibuilder on Linux there is nothing in /home/[user]/.local/share/vcmi

**Game logs**
[NOT REALLY GAME LOGS]
........
........
[unshield_reader_read:553] bytes_left = 0, volume_bytes_left = 5498
[unshield_reader_read:519] unshield_reader_read start: bytes_left = 0x157a, volume_bytes_left = 0x157a
[unshield_reader_read:531] Trying to read 0x157a bytes from offset 009bf666 in volume 3
[unshield_reader_read:553] bytes_left = 0, volume_bytes_left = 0
[unshield_file_save:759] read_bytes = 5498
[unshield_reader_read:519] unshield_reader_read start: bytes_left = 0x2, volume_bytes_left = 0x0
[unshield_reader_read:531] Trying to read 0x0 bytes from offset 009c0be0 in volume 3
[unshield_reader_read:535] bytes_to_read can't be zero
[unshield_file_save:720] Failed to read 2 bytes of file 26 (h3maped.exe) from input cabinet file 3
Failed to extract file 'h3maped.exe'.
[unshield_read_file_descriptor:47] File descriptor offset 27: 00001d68
[unshield_read_file_descriptor:63] Name offset:      00004890
[unshield_read_file_descriptor:64] Directory index:  00000000
[unshield_read_file_descriptor:65] Flags:            0004
[unshield_read_file_descriptor:66] Expanded size:    00050e65
[unshield_read_file_descriptor:67] Compressed size:  0003fc54
[unshield_read_file_descriptor:68] Data offset:      000ca69d
[unshield_reader_open_volume:311] Open volume 1
[unshield_reader_open_volume:408] First file index = 0, last file index = 21
[unshield_reader_open_volume:410] First file offset = 00000000, last file offset = 00121319
[unshield_reader_create:610] Trying next volume...
........
........
[unshield_reader_read:553] bytes_left = 0, volume_bytes_left = 0
[unshield_file_save:759] read_bytes = 1193
Error: failed to extract from Install Shield installer!
[END-OF-FILE]

**To Reproduce**
1. In terminal run vcmibuilder --cd1 /media/[user]/HEROES3 --cd2 /media/[user]/H3_DISK2

**Expected behavior**
Expecting data files to be copied to /home/[user]/.local/share/vcmi

**Actual behavior**
Nothing is copied, have errors in console

**Did it work earlier?**
Have no idea, trying vcmi for the first time.

**Version**
 - OS: Linux Ubuntu mate, 5.15.0-56-generic #62-Ubuntu SMP Tue Nov 22 19:54:14 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux 
 - Version: 1.1.1+ppa2~ubuntu22.04.1
 - Unshield: 1.4.

**Comment by rilian-la-te** [comment]
Maybe you have a broken CDs? Or it works for installing H3 in Wine?

**Comment by IvanSavenko** [comment]
Could also be bug in `unshield`
Or some version of H3 CD that we never supported in the first place.

Potential solutions:
Option 1 - try installing from CD using Wine or Windows OS
Option 2 - try to use `unshield` tool manually

After that copy required files (Data, Maps, Mp3 directories) to ~/.local/share/vcmi/

**Comment by guest73** [comment]
I tried Gog executable and it worked but that uses a different program (innoextract).
As for the unshield, I guess something is going wrong with it. I tried the 1-disc version of HOMM3 and it failed, but I was able to successfully install it on my retro PC with Win98.
The same with 2-discs variant, I've got 2 mdf files which I converted to iso's. They are perfectly mountable and readable(I have not tried them on my retro PC, but I guess there should be no problems).
So, as it is most likely unshield issue, I will close this task.
